lnwallet/channel.go

```diff
@@ -5173,6 +5205,12 @@ func NewUnilateralCloseSummary(chanState *channeldb.OpenChannel, signer input.Si
 			},
 			MaturityDelay: 0,
 		}
+
+		// If this is a tweakless commitment, then we can safely blank
+		// out the SingleTweak value as it isn't needed.
+		if tweaklessCommit {
```

Comment:
I think not setting it above and only setting the tweak in case of `!tweaklessCommit` is clearer. Then the legacy flow is the exception.